File: Frontend/pizza_mobile/lib/models/pedido.dart
class Pedido {
  final int clienteId;
  final double valor;
  final bool fiado;

  Pedido({
    required this.clienteId,
    required this.valor,
    required this.fiado,
  });
  Map<String, dynamic> toJson() => {
        "clienteId": clienteId,
        "valor": valor,
        "fiado": fiado,
      };
}

File: backend/PizzaErp.Api/Models/Pedido.cs
namespace PizzaErp.Api.Models;
public class Pedido
{
  public int Id {get; set;}
  public int ClienteId { get; set; }
  public int PizzaId { get; set; }
  public DateTime Data { get; set; } = DateTime.Now;

  public decimal Valor { get; set; }
  public bool Fiado {get; set;}
}